app/src/main/java/org/oppia/app/topic/ViewPagerAdapter.kt

```diff
@@ -0,0 +1,47 @@
+package org.oppia.app.topic
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import org.oppia.app.topic.overview.TopicOverviewFragment
+import org.oppia.app.topic.play.TopicPlayFragment
+import org.oppia.app.topic.review.TopicReviewFragment
+import org.oppia.app.topic.train.TopicTrainFragment
+
+/** Adapter to bind fragments to [FragmentStatePagerAdapter] inside [TopicFragment]. */
+class ViewPagerAdapter(fragmentManager: FragmentManager, private val topicId: String) :
+  FragmentStatePagerAdapter(fragmentManager) {
+
+  override fun getItem(position: Int): Fragment? {
+    val args = Bundle()
+    args.putString(TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY, topicId)
```

Comment:
I feel like we should use a key defined in this adapter rather than the activity one. We ought to have separate keys for arguments and intent values for each activity/fragment, but in this case I think it's reasonable to have a shared key for each of the tab fragments that's defined here to simplify things.

Reply:
Renamed when sent to fragment.